deployments/charts/kuma/templates/cp-webhooks-and-secrets.yaml

```diff
@@ -1,6 +1,9 @@
 {{- if not (eq (empty .Values.controlPlane.tls.general.caBundle) (empty .Values.controlPlane.tls.general.secretName)) }}
   {{ fail "You need to send both or neither of controlPlane.tls.general.caBundle and controlPlane.tls.general.secretName"}}
 {{- end }}
+{{- if and .Values.controlPlane.tls.general.certManager.enabled (or (not (empty .Values.controlPlane.tls.general.secretName)) (not (empty .Values.controlPlane.tls.general.caBundle))) }}
+  {{ fail "When controlPlane.tls.general.certManager.enabled is true, secretName and caBundle must be empty"}}
+{{- end }}
```

Comment:
Simplify the double negation:

```yaml
{{- if and .Values.controlPlane.tls.general.certManager.enabled (or .Values.controlPlane.tls.general.secretName .Values.controlPlane.tls.general.caBundle) }}
```

Helm's truthiness check handles empty strings, so `(not (empty ...))` is unnecessary.